Sources/Classes/core/Display/RealMiniAppView.swift

```diff
@@ -89,11 +89,11 @@ internal class RealMiniAppView: UIView {
     }
 
     fileprivate func initExternalWebViewClosures() {
-        onExternalWebviewResponse = { (url) in
-            self.webView.load(URLRequest(url: url))
+        onExternalWebviewResponse = { [weak self] (url) in
+            self?.webView.load(URLRequest(url: url))
         }
-        onExternalWebviewClose = { (url) in
-            self.didReceiveEvent(.externalWebViewClosed, message: url.absoluteString)
+        onExternalWebviewClose = { [weak self] (url) in
+            self?.didReceiveEvent(.externalWebViewClosed, message: url.absoluteString)
```

Comment:
did it capture `self` here as well? the others look more sus

Reply:
@laubengaier , yes, only when external webview is closed, however not mentioning this as weak reference wouldn't let the RealMiniAppView to call deinit method.

Reply:
ok good to know 👍